### ISSUE TITLE
Support reading in the terminal

### ISSUE THREAD

**Issue by orhun** [pythops/lobtui#2]
Instead of opening the browser 🐻

**Comment by pythops** [comment]
Definitely :100: 
Maybe the first version would be to convert html to markdown and render it in the terminal. 
I'm open for more suggestions :)

**Comment by orhun** [comment]
Sounds good!

**Comment by pythops** [comment]
maybe this what we need
https://github.com/letmutex/htmd

**Comment by orhun** [comment]
Yeah that looks good. Then we can pipe it into [`md-tui`](https://github.com/henriklovhaug/md-tui) probably.
